### PR TITLE
Add Pypi badge

### PR DESCRIPTION
**Pull request by NicolasGensollen**
I thought this could be useful. We can close this otherwise.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,9 @@
 	.. -*- mode: rst -*-
 
+.. image:: https://img.shields.io/pypi/v/nilearn.svg
+    :target: https://pypi.org/project/nilearn/
+    :alt: Pypi Package
+
```
Comment (tsalo) [inline README.rst:6]:
```suggestion
.. image:: https://img.shields.io/pypi/v/nilearn.svg
    :target: https://pypi.org/project/nilearn/
    :alt: Pypi Package

.. image:: https://img.shields.io/pypi/pyversions/nilearn.svg
    :target: https://pypi.org/project/nilearn/
    :alt: PyPI - Python Version
```

This is just a thought- but I think it's also useful to have a badge indicating which versions of Python for which the package is supported.

Comment (NicolasGensollen) [reply README.rst:6]: Great! thanks! :+1:

**Review by tsalo — APPROVED** [review]
I always love to see more badges!

**Review by bthirion — APPROVED** [review]
Great, thx !

**Comment by NicolasGensollen** [comment]
Thanks for the reviews! Merging...

**Pull request merged**